lib/cloud_controller.rb

```diff
@@ -113,6 +113,7 @@ module VCAP::CloudController; end
 
 require 'cloud_controller/route_binding_message'
 require 'cloud_controller/process_route_handler'
+require 'cloud_controller/route_handler'
```

Comment:
Note on naming:

slightly confusing to have `process_route_handler` and `route_handler`.

could we rename the latter to something that distinguishes it semantically? what does it handle routes for if not processes?